pkg/kernel/link.go

```diff
@@ -85,7 +86,7 @@ func (l *link) AddAddress(ip string) error {
 	}
 
 	// check if address is already assigned
-	current, err := netlink.AddrList(l.link, netlink.FAMILY_ALL)
+	current, err := netlink.AddrList(l.link, unix.AF_UNSPEC) // netlink.FAMILY_ALL
```

Comment:
Not clear why, could you add comments why this variant?

Reply:
`netlink` is a bit of linux-specific and all pretty constants such as `netlink.FAMILY_ALL`, `netlink.NUD_REACHABLE` are not available for OSx developer :)